test/test_parsing_files_with_commands.py

```diff
@@ -0,0 +1,29 @@
+from test.picardtestcase import PicardTestCase
+
+from picard.tagger import Tagger
+
+
+class TestParsingFilesWithCommands(PicardTestCase):
+    MOCK_FILE_CONTENTS = (
+        "FROM_FILE file0.mp3",
+        "",
+        "#commented command",
+        "LOAD file1.mp3 file2.mp3",
```

Comment:
Maybe also test a line with whitespaces at the start/end (for `strip()`).

Reply:
done :)